groups/config.yaml

```diff
@@ -0,0 +1,10 @@
+# Configuration for the kubernetes.io Google Groups setup
+
+# file path of the google groups service account token
+token-file: k8s-infra-test-project-1896690daeb3.json
+
+# Email id of the bot service account
+bot-id: wg-k8s-infra-api-test@kubernetes.io
```

Comment:
I thought @spiffxp wants to remove the account - do wee need it?

Reply:
yes we need it. we use this in the code around line 110

```
credential.Subject = config.BotID
```

and the subject field is

```
// Subject is the optional user to impersonate.
```